cpp/tests/io/text/data_chunk_source_test.cpp

```diff
@@ -331,4 +297,44 @@ TEST_F(DataChunkSourceTest, BgzipSourceVirtualOffsetsSingleChunk)
   test_source(input, *source);
 }
 
+TEST_F(DataChunkSourceTest, BgzipCompressedSourceVirtualOffsets)
```

Comment:
Was this test added because we now have `write_compressed_block` and `write_uncompressed_block`?

Reply:
exactly :)